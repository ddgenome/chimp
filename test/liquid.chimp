## liquid-phase only mechanism
mechanism "liquid.mech"
## parameter input
parameter "liquid.par"
## liquid-phase task
task "liquid.task"
