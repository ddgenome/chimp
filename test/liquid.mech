# a liquid-phase mechanism
A + B -> k_arrhenius(A1f, E1f) <- k_arrhenius(A1r, E1r) C;
C -> k_arrhenius(A2f, E2f) <- k_arrhenius(A2r, E2r) D;
C + D -> k_arrhenius(A3f, E3f) <- k_arrhenius(A3r, E3r) E;
