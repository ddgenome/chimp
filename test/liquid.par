# parameter input file for gas-phase mechanism
A1f	1.5e1	# m^3/mol/s
E1f	4.5e4	# J/mol
A1r	3.0e3	# 1/s
E1r	4.9e4	# J/mol
A2f	5.0e2	# 1/s
E2f	3.4e4	# J/mol
A2r	4.0e1	# 1/s
E2r	3.8e4	# J/mol
A3f	5.0e0	# m^3/mol/s
E3f	4.2e4	# J/mol
A3r	7.0e1	# 1/s
E3r	4.9e4	# J/mol
