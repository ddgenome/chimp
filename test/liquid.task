# -*- text -*-
# liquid-phase task input
begin model liquid
  output "liquid.out"
  begin integrator kmc
    scale 1.0e20
    begin state
      begin quantity
	c[A] = 1.0e3		# mol/m^3
	c[B] = 1.0e3		# mol/m^3
      end quantity
      begin output
	(1.0e2 5.0e2 1.0e2)
      end output
      begin reactor batch
	temperature 3.0e2	# K
	pressure 1.0e0		# Pa
	volume 1.0e0		# m^3
	rate_numerator moles
	rate_denominator volume
	fluid_quantity concentration
      end reactor
    end state
  end integrator
end model
