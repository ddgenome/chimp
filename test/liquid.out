# liquid
# x	A	B	C	D	E	steps
0.000000e+00	1.000000e+03	1.000000e+03	0.000000e+00	0.000000e+00	0.000000e+00	0
1.000003e+02	9.785925e+02	9.785925e+02	2.075825e+01	6.492712e-01	0.000000e+00	132829
2.000006e+02	9.580463e+02	9.580463e+02	3.950126e+01	2.449795e+00	1.328432e-03	267420
3.000010e+02	9.383194e+02	9.383194e+02	5.629015e+01	5.379486e+00	5.479783e-03	403911
4.000004e+02	9.195369e+02	9.195369e+02	7.121907e+01	9.214171e+00	1.494486e-02	540229
5.000005e+02	9.013903e+02	9.013903e+02	8.466198e+01	1.388394e+01	3.188237e-02	677836
